tests/pages/base.py

```diff
@@ -73,8 +73,8 @@ def open_navigation_menu(self, locator):
         def open_firefox_desktop_page(self):
             self.open_navigation_menu(self._firefox_menu_locator)
             self.find_element(*self._firefox_desktop_page_locator).click()
-            from .firefox.home import FirefoxHomePage
-            return FirefoxHomePage(self.selenium, self.page.base_url).wait_for_page_to_load()
+            from .firefox.new.download import DownloadPage
+            return DownloadPage(self.selenium, self.page.base_url).wait_for_page_to_load()
```

Comment:
I think you may need to add `params=''` here to stop the test erroring
```suggestion
            return DownloadPage(self.selenium, self.page.base_url, params='').wait_for_page_to_load()
```

Reply:
Thanks! Tests are re-running 🤞